spk/hassio/Makefile

```diff
@@ -1,11 +1,12 @@
 SPK_NAME = hassio
-SPK_VERS = 20191224
+SPK_VERS = 20200825
 SPK_REV = 1
 SPK_ICON = src/$(SPK_NAME).png
 SPK_DEPENDS = "Docker>=18.09.0-0506"
 DSM_UI_DIR = app
 
 DEPENDS =
+REQUIRED_DSM = 6.2 
```

Comment:
This is worth nothing, please remove it.

DSM 6.2 is no requirement for this package. Even for archs that have no toolchain available for DSM 6 or DSM 6.1 we can build packages. If we build for arch-geminilake-6.1 the build framework will auto choose the 6.2 toolchain.
The REQUIRED_DSM option sould be avoided anyway, as there will be conflicts wenn building arch-armv7-6.1 and arch-armv7-1.2

Reply:
It is only supported on DSM 6.2 and x64 archs..